crates/core/src/lib.rs

```diff
@@ -17,6 +23,150 @@ static mut JS_CONTEXT: OnceCell<Context> = OnceCell::new();
 static mut ENTRYPOINT: (OnceCell<Value>, OnceCell<Value>) = (OnceCell::new(), OnceCell::new());
 static SCRIPT_NAME: &str = "script.js";
 
+macro_rules! bind_js_return {
```

Comment:
Could you add comments to each of these macros explaining what they do, and if possible give an example of what they produce (if it's not too gnarly)?

Reply:
Done. I didn't add an example to the second one, but pointed to a function that is similar to the output.